<launch>
  <node pkg="static_map" type="static_map_node" name="static_map_node" output="screen">
    <param name="pcd_file" value="$(find bag_file)/pcd/map_0418_tf.pcd" />
    <param name="map_frame" value="map" />
    <param name="duration" value="1.0" />
    <param name="use_local_map" value="false" />
    <remap from="/static_map" to="/map/point_cloud" />
  </node>
  <node pkg="map_filter" type="rt_filter_node" name="rt_filter_node" output="screen">
    <remap from="/raw_pc" to="/map/point_cloud" />
    <param name="filter_xh" value="100" />
    <param name="filter_yh" value="100" />
    <param name="filter_zh" value="1.6" />
    <param name="filter_xl" value="-100" />
    <param name="filter_yl" value="-100" />
    <param name="filter_zl" value="0" />
    <param name="frame_id" value="/map" />
  </node>
  <node pkg="map_filter" type="filter_node" name="filter_node" output="screen">
    <remap from="/points_raw" to="/filtered_pc" />
    <param name="mean_k" value="12" />
    <param name="stddev" value="3.0" />
  </node>
</launch>